No TLS 1.3 support for GovCloud environments (#6631)
elastic/docs-content: deploy-manage/deploy/elastic-cloud/fedramp.md

% I'll single-source this if the finalized content is identical to what we have in the security section.

> Only FIPS 140-2 compliant TLS protocols, ciphers, and curve types are allowed to be used as listed below.

Author: ```suggestion
Only the following FIPS 140-2 compliant TLS protocols, ciphers, and curve types are allowed to be used:
```
